pages/networks/network1/resources.md

```diff
@@ -30,19 +29,33 @@ The above info can be found in this [`networks` repository](https://github.com/d
 | WS    | `wss://indexer.dydx.trade/v4/ws` |
 
 
+## StateSync nodes
+| Team           |  URI                                                                                  |
+|----------------|---------------------------------------------------------------------------------------|
+| KingNodes      |  |
+| Polkachu       |  |
+| Bware Labs     |  |
+| Lavender.Five  |  |
+| NodeStake      |  |
+| AutoStake      |  |
+
 ## Snapshot service
-| Type      | URI                                                    |
-|-----------|--------------------------------------------------------|
-| StateSync | `https://dydx-mainnet-statesync-rpc.bwarelabs.com` <br> `https://polkachu.com/state_sync/dydx` <br> `https://dydx-ops-statesync.kingnodes.com/` <br> `https://services.lavenderfive.com/mainnet/dydx/statesync` <br> `https://nodestake.top/dydx` <br> `https://autostake.com/networks/dydx/#services` <br> `https://genznodes.dev/resources/statesync/dydx` |
-| Snapshots | `https://bwarelabs.com/snapshots/dydx` <br> `https://polkachu.com/tendermint_snapshots/dydx` <br> `https://dydx-archive-snapshot.kingnodes.com/` <br> `https://services.lavenderfive.com/mainnet/dydx/snapshot` <br> `https://nodestake.top/dydx`  <br> `https://autostake.com/networks/dydx/#services` <br> `https://genznodes.dev/resources/snapshot/dydx` |
+| Team           | URI                                                       | Pruning |
```

Comment:
Please add another column for index. This is important for archives.
Kingnodes archive index `kv`.
All other snapshots are likely `null` but would need to confirm with operators.